model_cards/loodos/albert-base-turkish-uncased/README.md

```diff
@@ -0,0 +1,48 @@
+---
+language: tr
+---
+
+# Turkish Language Models with Huggingface's Transformers
+
+As R&D Team at Loodos, we release cased and uncased versions of most recent language models for Turkish. More details about pretrained models and evaluations on downstream tasks can be found [here](https://github.com/Loodos/turkish-language-models)
+
+# Turkish ALBERT-Base (uncased)
+
+This is ALBERT-Base model which has 12 repeated encore layers with 768 hidden layer size trained on uncased Turkish dataset.
+
+## Usage
+
+Using AutoModel and AutoTokenizer from Transformers, you can import the model as described below.
+
+```python
+from transformers import AutoModel, AutoTokenizer
+tokenizer = AutoTokenizer.from_pretrained("loodos/albert-base-turkish-uncased", do_lower_case=False, keep_accents=True)
+model = AutoModel.from_pretrained("loodos/albert-base-turkish-uncased") 
+
+normalizer = TextNormalization()
+normalized_text = normalizer(text, do_lower_case=True)
+tokenizer.tokenize(normalized_text)
+```
+
+### Notes on Tokenizers
+Currently, huggingface's tokenizers (which were written in Python) have a bug concerning letters "ı, i, I, İ" and non-ASCII Turkish specific letters. There are 2 reasons.
+
+1- Vocabulary and sentence piece model is created with NFC/NFKC normalization but tokenizer uses NFD/NFKD. NFD/NFKD normalization changes text that contains Turkish characters I-ı, İ-i, Ç-ç, Ö-ö, Ş-ş, Ğ-ğ, Ü-ü. This causes wrong tokenization, wrong training and loss of information. Some tokens are never trained.(like "şanlıurfa", "öğün", "çocuk" etc.) NFD/NFKD normalization is not proper for Turkish.
```

Comment:
is this also true with the FastTokenizers from https://github.com/huggingface/tokenizers?

cc @n1t0